test/commons/text/is-human-interpretable.js

```diff
@@ -0,0 +1,55 @@
+describe('text.isHumanInterpretable', function() {
+	it('returns 0 when no given string is empty', function() {
+		var actual = axe.commons.text.isHumanInterpretable('');
+		assert.equal(actual, 0);
+	});
+
+	it('returns 0 when given a single character that is blacklisted as icon', function() {
+		var actual = axe.commons.text.isHumanInterpretable('x');
+		assert.equal(actual, 0);
+	});
+
+	it('returns 0 when given a single character that is blacklisted is used as icon', function() {
+		var actual = axe.commons.text.isHumanInterpretable('i');
+		assert.equal(actual, 0);
+	});
+
+	it('returns 0 when given a single character that is a punctuation is used as icon', function() {
+		var actual = axe.commons.text.isHumanInterpretable('?');
+		assert.equal(actual, 0);
+	});
+
+	it('returns 0 when given a string has punctuations', function() {
+		var actual = axe.commons.text.isHumanInterpretable('I like 🏀');
+		assert.equal(actual, 0);
```

Comment:
This shouldn't return 0. Clearly this is text. Also, an emoji isn't a punctuation.